src/emulator/downloadableEmulators.ts

```diff
@@ -213,6 +213,11 @@ async function _runBinary(
     try {
       emulator.instance = childProcess.spawn(command.binary, command.args, {
         env: { ...process.env, ...extraEnv },
+        // `detached` must be true as else a SIGINT (Ctrl-c) will stop the child process before we can handle a
```

Comment:
Nice find!  My only concern is that on Windows this mentions something about a new console window:
https://nodejs.org/api/child_process.html#child_process_options_detached

Are you on Windows?  If not I can test this on my personal machine.  Want to make sure it's not popping up a new window for each emulator.

Reply:
Don't have a windows handy, please check :).

Reply:
No problem here on Windows.